extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -229,6 +230,7 @@ public function getAllAdminsData() {
 	 */
 	public function getRecentlyJoinedData() {
 		$recentlyJoined = $this->usersModel->getRecentlyJoinedUsers();
+		$recentlyJoined = $this->applyBadges( $recentlyJoined );
```

Comment:
could we rename this variable to reflect the change after function call or simply avoid creating it?